ContextualPopupDecorator/ContextualPopupDecorator.js

```diff
@@ -551,6 +553,9 @@ const Decorator = hoc(defaultConfig, (config, Wrapped) => {
 
 				this.calcOverflow(containerNode, clientNode);
 				this.adjustDirection();
+				if (this.props.onAdjustDirection) {
+					this.props.onAdjustDirection({adjusteddirection: this.adjustedDirection.split(' ')[0] === 'below' ? 'below' : 'above'});
```

Comment:
Could you please explain why we need to use a conditional operator here?

Reply:
ContextualPopup and ContextualPopupDecorator are used by other components, not only by Dropdown.
Error will occur if ContextualPopup is used by other components because they don't have `onAdjustedDirection` prop.

Reply:
Oh, the conditional operator that I asked was `adjusteddirection: this.adjustedDirection.split(' ')[0] === 'below' ? 'below' : 'above'`. Not the if statement.

Reply:
It seems we don't need conditional operator here.
I deleted it.